pkg/apis/istio/v1alpha1/values_types.proto

```diff
@@ -52,6 +52,26 @@ message ArchConfig {
 message CNIConfig {
   // Controls whether CNI is enabled.
   google.protobuf.BoolValue enabled = 1;
+
```

Comment:
Add a test to check that bad CNI field causes error, good one passes?

Reply:
added unit test here, would create separate PR for fully validation e2e test